schema/proto3/common.proto

```diff
@@ -25,8 +25,10 @@ enum ResourceType {
     VPC = 0;
     SUBNET = 1;
     PORT = 2;
-    SECURITYGROUP = 3;
-    DHCP = 4;
+    NEIGHBOR = 3;
```

Comment:
Resource type is a type for customer-defined resources. Could you elaborate why neighbor is a type?

Reply:
GoalStateOperationStatus includes ResourceType for GoalStateOperationReply. Since we introduced NeighborState in the schema, we need to add NEIGHBOR as ResourceType for our GoalStateOperationReply.